core/core.ts

```diff
@@ -695,7 +675,7 @@ export function fileGenerator(): FileGenerator {
   };
 }
 
-const VAR_REGEX = /\${([a-zA-Z0-9_.]+)}$/;
+const VAR_REGEX = /\${([a-zA-Z0-9-_.]+)}$/;
```

Comment:
Do we actually have minus signs in resource names?

Reply:
Yes. `zone_id` is the biggest offender on our side.